src/GraphQLParser.Tests/GraphQLParser.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>net6;net5;netcoreapp3.1</TargetFrameworks>
     <Nullable>disable</Nullable>
-    <NoWarn>$(NoWarn);1591</NoWarn>
+    <NoWarn>$(NoWarn);1591;IDE0008;IDE0022;IDE0058;IDE1006</NoWarn>
```

Comment:
Not important suggestions - use `Async` suffix, var/explicit, expression body, remove unnecessary expression value